src/schemas/generator/v2/thresholds.ts

```diff
@@ -0,0 +1,42 @@
+import { z } from 'zod'
```

Comment:
This Schema may evolve to include validation and potentially some conditional _transform_ operations.

It is based on the Thresholds feature available in the Test Builder.

<img width="952" alt="image" src="https://github.com/user-attachments/assets/91f75184-b714-42fc-833f-06fe7f131813" />